rascaline/Cargo.toml

```diff
@@ -36,7 +36,7 @@ name = "soap-power-spectrum"
 harness = false
 
 [dependencies]
-equistore = {git = "https://github.com/lab-cosmo/equistore", rev = "6c5a833", features = ["rayon"]}
+equistore = {git = "https://github.com/lab-cosmo/equistore", rev = "e5b9dc365369ba2584ea01e9d6a4d648008aaab8", features = ["rayon"]}
```

Comment:
I think the short hash would be enough but we will change it soon. lets keep it like this.